composer.json

```diff
@@ -7,6 +7,10 @@
         {
             "name": "Taylor Otwell",
             "email": "taylorotwell@gmail.com"
+        },
+        {
+            "name": "Tom de Goede",
+            "email": "tom@allesonline.eu"
```

Comment:
O so it keeps adding new commits to an already made pull request, didn't expect that. Thanks